.gitignore

```diff
@@ -88,3 +88,4 @@ reference/
 wikipathways-20210810-gmt-Homo_sapiens.gmt
 unify_temposeq_manifests/
 data/
+inputs/
```

Comment:
Ignoring everything in the inputs folder is going to stop us from updating the config.default.yaml file and the small README files that I added to each input folder. It's already ignoring all fastq files (lines 34-37) and the metadata, contrasts, and config files (lines 46, 47, 48). If there are other files you want ignores, can you add them to the gitignore more specifically?

Reply:
Good point! Thanks @LMBradford .